### DISCUSSION TITLE
大家有没有知道实体机怎么开服

### DISCUSSION THREAD

**Discussion by 0LYT0** [Meloong-Git/PCL#4800] [讨论]
大家有没有知道实体机怎么开服，我用网上的教程java报错了

**Comment by tangge233** [comment]
可以使用 MCSM 和 MCSL2（新手推荐） 等工具，以及……PCL 不支持开服，将来也不太会支持………………

**Comment by kndxhz** [comment]
市面上有很多一键开服工具，我也推荐MCSL2（Windows），如果是linux服务器建议使用MCSM
